directories/local-conformance-2_0.finsemble.apps.json

```diff
@@ -1,885 +1,940 @@
 {
-  "comment": "AppD component config",
```

Comment:
this file shouldn't have finsemble in the name - Finsemble uses the standard appD record now.